.github/workflows/node.js.yml

```diff
@@ -32,26 +32,6 @@ jobs:
               uses: GabrielBB/xvfb-action@v1
               with:
                   run: npm test
-            - name: Code coverage
```

Comment:
These are all moved to the new 'lint' job

Reply:
codecoverage must stay here because it depends on `npm test`